src/EDMF/apply_bcs.jl

```diff
@@ -54,3 +54,56 @@ function apply_bcs!(grid::Grid, q::StateVec, aux::StateVec, params, case::BOMEX)
     apply_Neumann!(q, :tke, grid, 0.0, (Zmin(), Zmax()), en)
     q[:tke, k_1, en] = surface_tke(ustar, wstar, zLL, obukhov_length)
 end
+
+
+function apply_bcs!(
+    grid::Grid,
+    q::StateVec,
+    tmp::StateVec,
+    params,
+    case::TRMM_LBA,
+)
+    @unpack params obukhov_length SurfaceModel wstar UpdVar
+    ustar = SurfaceModel.ustar
+    ρq_tot_flux = SurfaceModel.ρq_tot_flux
+    ρθ_liq_flux = SurfaceModel.ρθ_liq_flux
+    gm, en, ud, sd, al = allcombinations(q)
+    n_updrafts = length(ud)
+    k_1 = first_interior(grid, Zmin())
+    zLL = grid.zc[k_1]
+    θ_liq_1 = q[:θ_liq, k_1, gm]
+    q_tot_1 = q[:q_tot, k_1, gm]
+    alpha0LL = tmp[:α_0, k_1]
+    cv_q_tot = surface_variance(
+        ρq_tot_flux * alpha0LL,
+        ρq_tot_flux * alpha0LL,
+        ustar,
+        zLL,
+        obukhov_length,
+    )
+    cv_θ_liq = surface_variance(
+        ρθ_liq_flux * alpha0LL,
+        ρθ_liq_flux * alpha0LL,
+        ustar,
+        zLL,
+        obukhov_length,
+    )
+    @inbounds for i in ud
+        UpdVar[i].surface_bc.a = SurfaceModel.area / n_updrafts
```

Comment:
```suggestion
        UpdVar[i].surface_bc.a = bound(SurfaceModel.area / n_updrafts, params[:a_bounds])
```
should this be constrained by `a_bounds`?